app/assets/javascripts/student_profile/take_notes.jsx

```diff
@@ -97,6 +98,45 @@ export default React.createClass({
     return (noteInProgressType === null || !this.isValidAttachmentUrls());
   },
 
+  noteText() {
+    // In most cases, display the text the user has entered into the controlled
+    // text input, passed in as props. When the user is a counselor entering a
+    // High School Transition Note, show an initial prompt.
+
+    const {noteInProgressText, noteInProgressType} = this.props;
+
+    if (noteInProgressType !== 307) return noteInProgressText;
```

Comment:
Adding branching like this in a few different places is a good signal that the new thing might be somewhat different than the existing similar things (here, the transition packet and notes).  This can be a great path for building quickly, since it reuses other related code!  The tradeoff is that it's adding complexity across a few separate components rather than isolating it to a separate component or place.  I don't have a suggestion here, this just seems a good thing to be mindful of as you're iterating!